buildkite/resource_pipeline.go

```diff
@@ -295,9 +295,9 @@ func preparePipelineRequestPayload(d *schema.ResourceData) *Pipeline {
 	for k, vI := range d.Get("env").(map[string]interface{}) {
 		req.Environment[k] = vI.(string)
 	}
-	req.ProviderSettings = map[string]bool{}
+	req.ProviderSettings = map[string]string{}
```

Comment:
This field is still typed as `map[string]bool{}` in the struct, I think... so I don't think this can compile, can it?